core/src/com/unciv/ui/worldscreen/unit/UnitActions.kt

```diff
@@ -769,15 +767,15 @@ object UnitActions {
         val giftAction = {
             if (recipient.isCityState()) {
                 for (unique in unit.civInfo.getMatchingUniques("Gain [] Influence with a [] gift to a City-State")) {
-                    if((unit.isGreatPerson() && unique.params[1] == "Great Person")
-                        || unit.matchesFilter(unique.params[1])) {
-                        recipient.getDiplomacyManager(unit.civInfo).influence += unique.params[0].toInt() - 5
+                    if ((unit.isGreatPerson() && unique.params[1] == "Great Person")
```

Comment:
Don't great people have a "Great Person" unique? If not, then they probably should, the stat-based marking is obsolete